hotdoc/extensions/gst/gst_extension.py

```diff
@@ -784,62 +783,53 @@ def __create_property_symbols(self, obj, parent_uniquename,
             extra_content = self.formatter.format_flags(flags)
             res.extension_contents['Flags'] = extra_content
             if default:
-                if prop['type-name'] in ['GstCaps', 'GstStructure']:
+                if prop_type_name in ['GstCaps', 'GstStructure']:
                     default = '<pre class="language-yaml">' + \
                         '<code class="language-yaml">%s</code></pre>' % default
                 res.extension_contents['Default value'] = default
 
-    def __create_enum_symbol(self, type_name, enum, element_name, parent_name=None):
+    def __create_enum_symbol(self, type_name, enum, pagename, parent_name=None):
         display_name = re.sub(
             r"([a-z])([A-Z])", r"\g<1>-\g<2>", type_name.replace('Gst', ''))
+
         unique_name = type_name
         if self.app.database.get_symbol(unique_name):
-            unique_name = element_name + '_' + type_name
-        symbol = self.app.database.get_symbol(unique_name)
-        if not symbol:
-            members = []
-            for val in enum:
-                value_unique_name = "%s::%s" % (type_name, val['name'])
-                if self.app.database.get_symbol(value_unique_name):
-                    value_unique_name = "%s::%s" % (
-                        element_name + '_' + type_name, val['name'])
-                member_sym = self.create_symbol(GstNamedConstantValue,
-                                                unique_name=value_unique_name,
-                                                display_name=val['name'],
-                                                value=val['value'],
-                                                parent_name=parent_name, val=val,
-                                                extra={'gst-element-name': element_name})
-                if member_sym:
-                    members.append(member_sym)
-            symbol = self.create_symbol(
-                GstNamedConstantsSymbols, anonymous=False,
-                raw_text=None, display_name=display_name.capitalize(),
-                unique_name=unique_name, parent_name=parent_name,
-                members=members,
-                extra={'gst-element-name': 'element-' + element_name})
-        elif not isinstance(symbol, GstNamedConstantsSymbols):
-            self.warn('symbol-redefined', "EnumMemberSymbol(unique_name=%s, project=%s)"
-                      " has already been defined: %s" % (unique_name, self.project.project_name,
-                                                         symbol))
-
-            return None
+            # Still required as some bin install children properties inside
+            # themselves (like GstFakeSinkStateError in fakevideosink for example)
+            unique_name = pagename + '_' + type_name
```

Comment:
bwerg that's one ugly situation, fakevideosink should really implement GstChildProxy, and we should have that information in the cache, no?

Reply:
pls just update the comment to `Still required as some bins manually proxy their children's properties by reinstalling them at their level (see fakevideosink).`